src/ApiService/ApiService/Functions/Proxy.cs

```diff
@@ -16,7 +16,7 @@ public class Proxy {
         _context = context;
     }
 
-    [Function("proxy")]
+    [Function("Proxy")]
     public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "GET", "PATCH", "POST", "DELETE")] HttpRequestData req) {
```

Comment:
We'll need a route added here

Reply:
Why?

Reply:
Or maybe not if George is right the matching is case-insensitive.